### DISCUSSION TITLE
FUSE mount in the background

### DISCUSSION THREAD

**Discussion by NefixEstrada** [seaweedfs/seaweedfs#2135] [Q&A] [answered]
Hello!

First of all, thank you for this awesome software!! I've a question: is it possible to run `weed mount` in the background? It would be really useful, since it would enable using the filesystem in the `/etc/fstab` and mounting it automatically on boot. Thanks! :)

**Comment by chrislusf** [comment]
cc @danielflira 

with these? https://github.com/chrislusf/seaweedfs/wiki/FUSE-Mount

check `weed fuse -h` also

```
Or add weed as a mount subtype:

cp weed /sbin/mount.weed
mount -t weed fuse /mnt -o "filer=localhost:8888,filer.path=/"
```

**Reply by chrislusf** [reply]
If figured out, welcome to update the wiki page to add more exact steps.

**Reply by NefixEstrada** [reply]
Oh, maybe I haven't explained myself correctly. I have successfully added a SeaweedFS mount in the `/etc/fstab`, but when I run `mount -a`, it keeps the command running in the foreground and if I `Ctrl+C` then it stops it. What I want is to be able to `mount` it and then keep it running in the background. If it runs in the foreground it is going to stuck the whole boot process, since the `mount` command is never going to end. What I have in my fstab is the following:

```
# fuse /opt/seaweedfs   fuse.weed       filer=192.168.1.10:8888,filer.path=/      0 2
```

**Reply by danielflira** [reply]
to explain "weed fuse" just parse parameters that "mount" sends to weed then call "weed mount" routine.

I don't know how "weed mount" works but I will try change to be possible to run in background.

**Reply by chrislusf** [reply]
@danielflira there are some daemonize packages that you may want to use.

**Comment by chrislusf** [comment, marked as answer]
same as #2137
